src/adapter/src/coord/sequencer.rs

```diff
@@ -2199,7 +2207,7 @@ impl<S: Append + 'static> Coordinator<S> {
         }
     }
 
-    fn plan_peek(
+    async fn plan_peek(
```

Comment:
Not thrilled that this an explain are now async. Are these getting fixed (either reverted, or else they now get a `tx` and handle responses themselves) in followups?

Reply:
Yeah I'm hoping to remove this in followups, I've added a TODO.